src/components/settings/Settings.h

```diff
@@ -3,6 +3,7 @@
 #include <bitset>
 #include "components/brightness/BrightnessController.h"
 #include "components/fs/FS.h"
+#include "displayapp/WatchFaces.h"
```

Comment:
other setting enums are defined in `Settings.h`. Do you think it would be a good idea to move the `WatchFace` enum into `Settings.h` as well? Just a comment. I'd like to hear your thoughts on this

Reply:
No, because I see watch faces more like apps.